wrappers/golang/build.sh

```diff
@@ -57,4 +91,17 @@ do
   echo "DEVMODE=${DEVMODE}" >> build_config.txt
   cmake -DCMAKE_CUDA_COMPILER=$CUDA_COMPILER_PATH -DCURVE=$CURVE -DG2=$G2_DEFINED -DECNTT=$ECNTT_DEFINED -DDEVMODE=$DEVMODE -DCMAKE_BUILD_TYPE=Release -S . -B build
   cmake --build build -j8 && rm build_config.txt
-done
\ No newline at end of file
+done
+
+# Needs to remove the CMakeCache.txt file to allow building fields after curves
+# have been built since CURVE and FIELD cannot both be defined
+rm -f "$BUILD_DIR/CMakeCache.txt"
+
+for FIELD in "${BUILD_FIELDS[@]}"
+do
+  echo "FIELD=${FIELD}" > build_config.txt
+  echo "DEVMODE=${DEVMODE}" >> build_config.txt
+  cmake -DCMAKE_CUDA_COMPILER=$CUDA_COMPILER_PATH -DFIELD=$FIELD -DEXT_FIELD=$EXT_FIELD -DDEVMODE=$DEVMODE -DCMAKE_BUILD_TYPE=Release -S . -B build
+  cmake --build build -j8 && rm build_config.txt
```

Comment:
`-j8` => `-j`

Reply:
Why do we want to use the build tool's default number of jobs?

Reply:
What's the default number here?
Maybe it's better to set it to use all the cpu cores

Reply:
I actually thought that -j will use num-cpus threads

Reply:
https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-build-j. Lets leave as is for now unless we have a reason to change it